lighthouse-core/audits/byte-efficiency/total-byte-weight.js

```diff
@@ -60,7 +73,7 @@ class TotalByteWeight extends ByteEfficiencyAudit {
       const result = {
         url: record.url,
         totalBytes: record.transferSize,
-        totalMs: ByteEfficiencyAudit.bytesToMs(record.transferSize, networkThroughput),
+        flagged: TotalByteWeight.hasExceededJSBundleSize(record),
```

Comment:
Would kinda prefer to only add the boolean in the true case. @brendankenny type-wise is that OK?